1036/en.md

```diff
@@ -0,0 +1,185 @@
+# LOJ 1036 - A Refining Company
+
+---
+Tags : Dynamic Programming, Best Cumulative Sum, Memoization
+
+The problem statement has these key points:
+
+1. Conveyor belts moving row-wise in the same column from block __b__ to the __radium refinery__ can be added up as long as there is no column-wise conveyor belt moving in between them.
+2. Conveyor belts moving column-wise in the same row from block __b__ to the __uranium__ can be added up as long as there is no row-wise conveyor belt moving in between them.
+3. Any kind of conveyor is NOT _bi-directional_ when it comes to collection. It has to reach the refinery, in one direction. Either from East to West or from North to South, BUT West to East or South to North is not possible as the refineries’ placement is already given. This plays a heavy role in how to implement and memoize row-wise (left-to-right or right-to-left?) and column-wise (up-to-down or down-to-up?).
```

Comment:
"row-wise" and "column-wise" seem confusing.
Particularly I am very confused by "Conveyor belts moving row-wise in the same column" and "Conveyor belts moving column-wise in the same row".

Reply:
Resolved by mentioning `left-->right` and `up-->down`.